services/web-app/utils/sendLocalRequest.js

```diff
@@ -37,7 +38,8 @@ export default function sendLocalRequest(
   body = null,
   requestOptions = {}
 ) {
-  const protocol = process.env.RUNNING_SECURELY === '1' ? 'https' : 'http'
+  const protocol =
+    getRunMode() === RunMode.Standard || process.env.RUNNING_SECURELY === '1' ? 'https' : 'http'
```

Comment:
If we can get TLS termination exclusively in the proxy-server.js file, then internal requests could go back to assuming they're always using http.

Reply:
Done!! 🎉